systemtests/src/main/java/com/github/streamshub/systemtests/constants/ResourceKinds.java

```diff
@@ -6,37 +6,41 @@ private ResourceKinds() {}
-    public static final String CONFIG_MAP = "configmap";
-    public static final String DEPLOYMENT = "deployment";
-    public static final String DEPLOYMENT_CONFIG = "deploymentconfig";
-    public static final String POD = "pod";
-    public static final String SERVICE = "service";
-    public static final String SECRET = "secret";
-    public static final String CLUSTER_ROLE = "clusterrole";
-    public static final String CLUSTER_ROLE_BINDING = "clusterrolebinding";
-    public static final String ROLE_BINDING = "rolebinding";
-    public static final String SERVICE_ACCOUNT = "serviceaccount";
-    public static final String JOB = "job";
-    public static final String NAMESPACE = "namespace";
-    // ----------
-    // Openshift
-    // ----------
-    public static final String CATALOG_SOURCE = "catalogsource";
-    public static final String SUBSCRIPTION = "subscription";
-    public static final String INSTALL_PLAN = "installplan";
-    public static final String CLUSTER_SERVICE_VERSION = "clusterserviceversion";
-    public static final String OPERATOR_GROUP = "operatorgroup";
-    public static final String OPERATOR = "operator";
+    public static final String CUSTOM_RESOURCE_DEFINITION = "CustomResourceDefinition";
+    public static final String CONFIG_MAP = "ConfigMap";
+    public static final String DEPLOYMENT = "Deployment";
+    public static final String DEPLOYMENT_CONFIG = "DeploymentConfig";
+    public static final String POD = "Pod";
+    public static final String SERVICE = "Service";
+    public static final String SECRET = "Secret";
+    public static final String CLUSTER_ROLE = "ClusterRole";
+    public static final String CLUSTER_ROLE_BINDING = "ClusterRoleBinding";
+    public static final String ROLE_BINDING = "RoleBinding";
+    public static final String SERVICE_ACCOUNT = "ServiceAccount";
+    public static final String JOB = "Job";
+    public static final String NAMESPACE = "Namespace";
+
+    // ----------
+    // OpenShift
+    // ----------
+    public static final String CATALOG_SOURCE = "CatalogSource";
+    public static final String SUBSCRIPTION = "Subscription";
+    public static final String INSTALL_PLAN = "InstallPlan";
+    public static final String CLUSTER_SERVICE_VERSION = "ClusterServiceVersion";
+    public static final String OPERATOR_GROUP = "OperatorGroup";
+    public static final String OPERATOR = "Operator";
+
     // ----------
     // Kafka
     // ----------
-    public static final String KAFKA_TOPIC = "kafkatopic";
-    public static final String KAFKA = "kafka";
-    public static final String KAFKA_USER = "kafkauser";
-    public static final String KAFKA_NODE_POOL = "kafkanodepool";
+    public static final String KAFKA_TOPIC = "KafkaTopic";
+    public static final String KAFKA = "Kafka";
+    public static final String KAFKA_USER = "KafkaUser";
+    public static final String KAFKA_NODE_POOL = "KafkaNodePool";
```

Comment:
You can use the RESOURCE_KIND from each of the resource from the Strimzi's API module.